code/datums/loadout/item_representation/_item_representation.dm

```diff
@@ -66,19 +66,52 @@
 	for(var/atom/thing_in_content AS in item_to_copy.contents)
 		if(!isitem(thing_in_content))
 			continue
-		if(!is_savable_in_loadout(thing_in_content))
-			continue
 		item_representation_type = item2representation_type(thing_in_content.type)
 		contents += new item_representation_type(thing_in_content)
 
-/datum/item_representation/storage/instantiate_object(datum/loadout_seller/seller, master = null)
+/datum/item_representation/storage/instantiate_object(datum/loadout_seller/seller, master = null, mob/living/user)
 	. = ..()
 	if(!.)
 		return
 	var/obj/item/storage/storage = .
+	var/list/obj/item/starting_items = list()
+	for(var/obj/item/I AS in storage.contents)
+		starting_items[I.type] = starting_items[I.type] + 1
+	storage.delete_contents()
```

Comment:
Here, we are saving what items where already in the storage when initialisating, so they can bypass vendor later